docs/core/install/windows.md

```diff
@@ -32,6 +32,8 @@ Windows 10 versions end-of-service dates are segmented by edition. Only **Home**
 
 | Operating System            | .NET Core 2.1 | .NET Core 3.1 | .NET 5 |
 |-----------------------------|---------------|---------------|--------|
+| Windows 11 (Preview) / Windows Server 2022  | ❌           | ✔️            | ✔️    |
```

Comment:
Windows 11 and Server 2022 are not related (different sources, they're not client/server releases for the same OS) so these should be on separate rows.